tests/unit_tests/test_psf_analysis.py

```diff
@@ -12,7 +12,7 @@
 
 def test_reading_simtel_file(args_dict, io_handler):
     test_file = io_handler.get_input_data_file(
-        file_name="photons-North-LSTN-01-d10.0-za20.0-off0.000_validate_optics.lis.gz",
+        file_name="photons-North-LSTN-01-d10.0km-za20.0deg-off0.000deg_validate_optics.lis.gz",
```

Comment:
Why do we use here `0.000deg` for 0 offset, but above ` 000deg ` for 0 azimuth angle?

Reply:
This is motivated by:

- zenith and offset angle needs to be set on a sub-degree accuracy
- azimuth not - it is probably impossible to see a difference between a shower coming from 180 or 185 deg. 

I am generally not a friend of putting metadata in the file name. This information should be in a metdata header. We should fix this at some point in the future.

Reply:
Yes this makes sense, but do we really need milli degree accuracy for offset? Agree to put this in the metadata in the future.

Reply:
We actually have simulated at 0.125 deg offset at some point, so this is not completely wrong.

Reply:
Ok thanks, everything is clear then. Going to mark it as approved.